app/controllers/admin/users_controller.rb

```diff
@@ -1,7 +1,9 @@
 class Admin::UsersController < ApplicationController
 
   before_filter :authenticate_user!
-  before_action -> { user_has_position(User.secretary) }
+  before_action -> { user_has_position(Position.first) }
```

Comment:
It seems like user_has_position should take a position name. I think this would be better if it was like `user_has_position(some_position_name)`